org.freedesktop.dabrain34.squared.json

```
@@ -0,0 +1,41 @@
+{
+  "app-id": "org.freedesktop.dabrain34.squared",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "squared",
+  "finish-args": [
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--share=ipc",
+    "--socket=x11",
+    "--persist=.local/share/godot/app_userdata/squared"
+  ],
+  "modules": [
+    {
+      "name": "squared",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://gitlab.freedesktop.org/dabrain34/squared/uploads/6af5657460b78743bdf42e4b9e493e1b/squared-0.1.0.tar.xz",
+          "sha256": "34757a44778f21a2033460edca6d56d5cbb44caa7b51d7436e63100c61e0341f"
+        },
+        {
+          "type": "file",
+          "url": "https://gitlab.freedesktop.org/dabrain34/squared/uploads/f52198b93be7f495a85c9f8e70c34c5e/squared_x86_64",
+          "sha256": "fd672ed5053a1a52998fa9a256347a2d91684a29fc061e4ba2d65c10919aa507"
+        },
+        {
+          "type": "file",
+          "url": "https://gitlab.freedesktop.org/dabrain34/squared/uploads/5fe7bf6de6f1d7a3c782822fd6b63732/data.pck",
+          "sha256": "5c47b5b0555de0a511886ff2272c5b4fc3b8a4c3422df79005de9d3b4d448cb5"
+        }
+      ],
+      "post-install": [
+        "install -Dm755 squared_x86_64 $FLATPAK_DEST/bin/squared",
+        "install -Dm555 data.pck $FLATPAK_DEST/bin/data.pck"
```

```suggestion
        "install -Dm755 ../squared_x86_64 $FLATPAK_DEST/bin/squared",
        "install -Dm555 ../data.pck $FLATPAK_DEST/bin/data.pck"
```
Will fix https://github.com/flathub/flathub/pull/4600/files#r1358342456